types/trusted-types/test/browser.ts

```diff
@@ -1,11 +1,3 @@
-import TT from 'trusted-types';
-
-// $ExpectType TrustedTypePolicyFactory
-TT;
-
-// $ExpectError
```

Comment:
please keep this one.

Reply:
Ok.

Reply:
resolve the comment but forgot to update?

Reply:
I am sure I was adding this, maybe I messed up something and reverted it. Thanks for noticing.